ansible_collections/arista/avd/plugins/module_utils/eos_designs_facts.py

```diff
@@ -301,15 +301,15 @@ def _switch_data(self):
 
         Vars are inherited like:
         <node_type_key>.defaults ->
-            <node_type_key>.node_groups.<group> ->
-                <node_type_key>.node_groups.<group>.nodes.<node> ->
-                <node_type_key>.nodes.<node>
+            <node_type_key>.node_groups.[<node_group>] ->
+                <node_type_key>.node_groups.[].nodes.[<node>] ->
```

Comment:
What does the empty `[]` mean, that it can be in any `node_group`?

Reply:
adding <node_group> inside the [].